packages/core/src/lib/select/searchable-select.svelte

```diff
@@ -405,7 +405,12 @@ const handleKeydown = createHandleKey({
               />
             {/if}
             <div class="flex flex-col">
-              <p class="text-wrap text-sm">
+              <p
+                class={cx(
+                  'text-wrap',
+                  option.description ? 'text-sm' : 'text-xs'
```

Comment:
If it's the fancy version show text-sm
<img width="378" alt="image" src="https://github.com/user-attachments/assets/93660f31-6723-41b3-8ef5-4407e4240340">

Reply:
Yea not sure if this is correct @nathartman